backend/dataall/modules/maintenance/api/enums.py

```diff
@@ -0,0 +1,18 @@
+"""Contains the enums used in maintenance module"""
+
```

Comment:
For graphql we use GraphQLEnum class tha maps enums to what we get from graphql. 

Here is an example in the notebooks module
```python
from dataall.base.api.constants import GraphQLEnumMapper


class SagemakerNotebookRole(GraphQLEnumMapper):
    """Describes the SageMaker Notebook roles"""

    CREATOR = '950'
    ADMIN = '900'
    NO_PERMISSION = '000'
```

Reply:
Used the `GraphQLEnumMapper ` instead of Enum